llama-index-integrations/multi_modal_llms/llama-index-multi-modal-llms-huggingface/README.md

```diff
@@ -0,0 +1,84 @@
+# LlamaIndex Multi_Modal_Llms Integration: Huggingface
+
+This project integrates Hugging Face's multimodal language models into the LlamaIndex framework, enabling advanced multimodal capabilities for various AI applications.
+
+## Features
+
+- Seamless integration of Hugging Face multimodal models with LlamaIndex
+- Support for multiple state-of-the-art vision-language models and their **finetunes**:
+  - [Qwen2 Vision](https://huggingface.co/collections/Qwen/qwen2-vl-66cee7455501d7126940800d)
+  - [Florence2](https://huggingface.co/collections/microsoft/florence-6669f44df0d87d9c3bfb76de)
+  - [Phi-3.5 Vision](https://huggingface.co/collections/microsoft/phi-3-6626e15e9585a200d2d761e3)
+  - [PaLI-Gemma](https://huggingface.co/collections/google/paligemma-release-6643a9ffbf57de2ae0448dda)
+- Easy-to-use interface for multimodal tasks like image captioning and visual question answering
+- Configurable model parameters for fine-tuned performance
+
+---
+Author of that Integration [GitHub](https://github.com/g-hano) | [LinkedIn](https://www.linkedin.com/in/chanyalcin/) | [Email](mcihan.yalcin@outlook.com)
+---
+
+## Installation
+
+```bash
+pip install llama-index transformers torch pillow
+```
+
+Make sure to set your Hugging Face API token as an environment variable:
+
+```bash
+export HF_TOKEN=your_huggingface_token_here
+```
+
+## Usage
+
+Here's a basic example of how to use the Hugging Face multimodal integration:
+
+```python
+from llama_index.multi_modal_llms import HuggingFaceMultiModal

```

Comment:
I think the real import is `from llama_index.multi_modal_llms.huggingface import HuggingFaceMultiModal`

Reply:
ye you are right, I fixed the issue. sorry its my very first time contributing.